### ISSUE TITLE
Regain access to the original repo (rmagick/rmagick)

### ISSUE THREAD

**Issue by vassilevsky** [rmagick/rmagick#205]
Looks like GitHub made their move. https://github.com/rmagick is now an organization. It has no repositories yet. @wurde and I are owners.

The old https://github.com/rmagick has been renamed to https://github.com/rmagick-temp . It has the old repository.

I'm not sure how to proceed from here.

Ideally we need to move the original repo (https://github.com/rmagick-temp/rmagick) to the new https://github.com/rmagick organization, merge this repo (https://github.com/gemhome/rmagick) onto it, and continue working on it. But I don't think we have permissions to do that.

@bf4 how is your conversation with Moncef going?

@ioquatix @u338steven @mockdeep @linduxed bringing you here to let you know about this development

**Comment by linduxed** [comment]
I don't see any reason for the previous repo to exist. This repository is a straight up continuation of the work, it didn't "branch away" from upstream at any point.

While I'm not sure how @mmaiza would feel about it (or you guys for that matter) but to minimize confusion for the end-users of this gem, I think we should just add a tag indicating the point at which the fork was created and move on. I see no reason why any work should be done on the pre-fork-rmagick or how anyone would benefit from having two projects which both are in some sense "the official version".

This project has moved far beyond what it was late July last year (when the fork happened) and continues to improve with the help of both the people mentioned in this issue, but also various sporadic contributors.

In short:

1. Move this repository to `rmagick/rmagick`.
2. Either delete or forget the old repository. I don't think the end users benefit in any way from having the old project around. If @mmaiza doesn't want to delete it, fine, it'll stay as one of the many forks of the project, but I don't see the reason why it should.

**Comment by vassilevsky** [comment]
git-wise, this repo is indeed just a continuation of the old one. It has all the history and tags.

What bothers me are issues and pull requests open at the old repo. It would be nice to properly close them. They are the originals and this repo's issues are mostly copies.

**Comment by linduxed** [comment]
With that said, all of the creators of the original issues were pinged when the issues were imported by @bf4. It would be *nice* to just move the old issues here and merge them manually into a nice cohesive whole, but that's something I think neither will nor *can* happen.

I don't think this is a problem.

**Comment by bf4** [comment]
We could just move gemhome/rmagick to rmagick/rmagick and leave the
original repo alone as a reference :)

**Comment by vassilevsky** [comment]
As you say.

**Comment by vassilevsky** [comment]
I have moved the repository and updated all links in the code.

**Comment by bf4** [comment]
wee!!!

**Comment by bf4** [comment]
@vassilevsky dunno if you want to re-add me as owner/collaborator to the rmagick org. 

Unrelated, let's fork https://github.com/minimagick/minimagick to rmagick and invite them to move here?

**Comment by wurde** [comment]
Congratulations guys :+1: this is well deserved.

**Comment by vassilevsky** [comment]
@bf4 check your email for an invitation.

@wurde thank YOU for your super-effective help!!!

Everyone, feel free to email me your postal address and I will send you postcards!

**Comment by vassilevsky** [comment]
Not sure about **minimagick**. What are potential benefits?

**Comment by vassilevsky** [comment]
@linduxed this seems to be a good time to post another writeup :) Your [first](http://linduxed.com/blog/2014/07/25/the-state-of-rmagick-development/) one was very helpful. It made it to Ruby Weekly! I saw it there and joined the project because of it :)

**Comment by mockdeep** [comment]
I wish we could get rid of that " forked from rmagick-temp/rmagick" message...

**Comment by bf4** [comment]
the only way for mortals to do it is the delete the repo and repush.. might need to ask github to help so we don't lose the issues

**Comment by bf4** [comment]
@vassilevsky great idea re: writeup!  

btw, if anyone wants to help set up an rmagick github page I'd be happy to help migrate it from the current site

re: minimagick just because they're related projects.  nothing more

**Comment by u338steven** [comment]
Cool!!!

@mockdeep @bf4
I hope this will be of some help.

[Delete fork dependency of a GitHub repository - Stack Overflow](http://stackoverflow.com/questions/16052477/delete-fork-dependency-of-a-github-repository)

> You can also ask support to switch your repository to "normal mode".
> 
> On [this page](https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile), "Commit was made in a fork" paragraph, it is explained that one has to go through support to switch. Therefore, it is likely that there is no way to do that by yourself (unless you destroy and recreate your repo which is explained before... if you do so be careful if you have tickets or a wiki attached to your project as they will be deleted!).

**Comment by bf4** [comment]
@rmagick/owners @u338steven behold: https://rmagick.github.io/ 

I just asked the imagemagick folks to send me a zip of the contents. It's now at https://github.com/rmagick/rmagick.github.io

> > Any chance we could just get a zip of http://www.imagemagick.org/RMagick/ and
then set http://www.imagemagick.org/RMagick/ to redirect or link to https://rmagick.github.io/ 

> Let us know when its ready and we'll add a redirect link @ that time.

So, if we're all happy, I'll tell them to set up the redirect.

**Comment by vassilevsky** [comment]
:+1:

**Comment by bf4** [comment]
Email sent

**Comment by linduxed** [comment]
@vassilevsky While it's more than a month since you asked, here you go!

http://linduxed.com/blog/2015/07/19/rmagick-a-year-later/

**Comment by bf4** [comment]
So, you could even add this to rmagick.github.io if you wish on a new 'blog' or 'changes' page :)

**Comment by vassilevsky** [comment]
Thanks Mikael! 💪

It's also worth mentioning that Benjamin raised the issue of abandoned gems in RubyGems issue tracker and there was a discussion and as a result a RailsGirls Summer Of Code team is now working on it:

https://github.com/rubygems/rubygems.org/issues/725

Maybe it's worth mentioning? IDK.

**Comment by vassilevsky** [comment]
I have submitted it to RubyFlow:

http://www.rubyflow.com/p/98jsja-rmagick-a-year-later

I hope Peter Cooper will include it in the next issue of Ruby Weekly.

**Comment by ioquatix** [comment]
Awesome work guys. Can't help but feel like I should correct history a bit. After learning about @bf4's ruby gem home, I asked him to get involved in this project, and spent a considerable amount of time tracking the initial issues and PRs, but once other people got involved I took a back seat.

**Comment by ioquatix** [comment]
Here is the original discussion which lead to me getting @bf4 involved in RMagick if memory serves me correctly: https://groups.google.com/forum/#!searchin/rubygems-org/rmagick/rubygems-org/niS5ZO9DNgk/3CeTTZcq770J

**Comment by ioquatix** [comment]
By the way, credit where credit is due :D

**Comment by vassilevsky** [comment]
http://rubyweekly.com/issues/256

I count this as a success :D
